app/controllers/application_controller.rb

```diff
@@ -27,7 +27,20 @@ def handle_access_denied(exception)
   helper_method :current_user
 
   def current_user
-    @current_user ||= session[:auth_user] if session[:auth_user]
+    if session[:auth_user]
+      @current_user ||= session[:auth_user]
+    elsif development_mode_auth?
+      email = authenticate_with_http_basic do |user, pass|
+        user if authorise_development_mode?(user, pass)
+      end
+      if email
+        setup_development_mode_session(
+          email: Settings.authorised_user.email,
+          first_name: Settings.authorised_user.first_name,
+          last_name: Settings.authorised_user.last_name,
+        )
+      end
```

Comment:
else raise hell as its a no op?

Reply:
No, I don't understand all the intricacies here, but if I raise an error it blocks the login attempt. I think it has to fall out of here and hit `request_http_basic_authentication` below to actually request the l/p.

Reply:
ok... i thought it will put someone out of their misery...